perfscale_regression_ci/utils/ocp_utils.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+
+#set -ex
+
+cluster_health_check() {
+    echo "Basic cluster health check and resource usage."
+    set -x
+    oc get machinesets -A
+    oc get nodes
+    oc get co --no-headers | grep -v 'True.*False.*False' || true
+    oc get pods --no-headers -A | egrep -v 'Running|Completed' | egrep -v installer | wc -l || true
+    oc adm top nodes --sort-by='cpu' | head -11
+    oc adm top nodes --sort-by='memory' | head -11
+    set +x
+}
+
+install_dittybopper() {
```

Comment:
This could also be moved to [common.sh here](https://github.com/openshift/svt/blob/master/perfscale_regression_ci/scripts/common.sh)